### ISSUE TITLE
Unit tests fail when run on simulator with Accessibility Bold Text enabled

### ISSUE THREAD

**Issue by mpospese** [yml-org/YMatterType#68]
Unit tests should pass regardless of the current configuration of the system.

The failures are due to using `DefaultFontFamily` with a font family and weight where the next heaviest weight (requested when accessibility bold text is enabled) is not installed. Failures happen for NotoSans (only .regular weight installed) and AvenirNext.bold (extra bold is installed but named "Heavy" and so DefaultFontFamily fails to load it because by default it looks for a font file named "ExtraBold").

We should use custom `FontFamily` implementations for both NotoSans and AvenirNext and specify both the supported weights and (in case of Avenir) override `weightName(for weight:)`.
